build.sbt

```diff
@@ -76,6 +76,7 @@ lazy val faciaJson = project.in(file("facia-json"))
     organization := "com.gu",
     name := "facia-json",
     resolvers ++= Seq(
+      Resolver.sonatypeRepo("releases"),
```

Comment:
What does this do?

Reply:
When we publish libraries, they're available on Sonatype repos first, then on Maven. This adds the former, so we don't have to wait hours before seeing the new versions live.